tests/validation/tests/v3_api/test_node_annotation.py

```diff
@@ -0,0 +1,554 @@
+    client, node = \
+        get_node_details(cluster_detail["cluster"], cluster_detail["client"])
+    node_annotations = node.annotations.data_dict()
+
+    # get user token and client
+    token = rbac_get_user_token_by_role(role)
+    print("token: ", token)
+    user_client = get_client_for_token(token)
+
+    node_annotations[annotation_key] = annotation_value
+
+    if role == CLUSTER_OWNER:
+        user_client.update(node, annotations=node_annotations)
+        time.sleep(2)
+
+        # annotation should be added
+        validate_annotation_set_on_node(user_client, node,
+                                        annotation_key,
+                                        annotation_value)
+    else:
+        with pytest.raises(ApiError) as e:
+            user_client.update(node, annotations=node_annotations)
+        assert e.value.error.status == 403
+        assert e.value.error.code == 'Forbidden'
+
+
+@if_test_rbac
+@pytest.mark.parametrize("role", roles)
+def test_rbac_node_annotation_add_kubectl(role):
+    annotation_key = random_name()
+    annotation_value = random_name()
+    # get node details
+    client, node = \
+        get_node_details(cluster_detail["cluster"], cluster_detail["client"])
+    node_name = node.nodeName
+
+    # get user token and client
+    token = rbac_get_user_token_by_role(role)
+    user_client = get_client_for_token(token)
+    print(cluster_detail["cluster"]["id"])
+    print(cluster_detail["cluster"])
+    cluster = user_client.list_cluster(id=cluster_detail["cluster"]["id"]).data
+    print(cluster)
+    create_kubeconfig(cluster[0])
+
+    # add annotation on node
+    command = "annotate nodes " + node_name + " " + \
+              annotation_key + "=" + annotation_value
+
+    if role == CLUSTER_OWNER:
+        execute_kubectl_cmd(command, False)
+        time.sleep(2)
+        # annotation should be added
+        validate_annotation_set_on_node(user_client, node,
+                                        annotation_key,
+                                        annotation_value)
+    elif role == CLUSTER_MEMBER:
+        result = execute_kubectl_cmd(command, False, stderr=True)
+        result = result.decode('ascii')
+        assert "cannot patch resource \"nodes\"" in result
+        assert "forbidden" in result
+    else:
+        result = execute_kubectl_cmd(command, False, stderr=True)
+        result = result.decode('ascii')
+        assert "cannot get resource \"nodes\"" in result
+        assert "forbidden" in result
+
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    cluster_detail["client"], cluster_detail["cluster"] = \
+        get_user_client_and_cluster()
+
+
+def check_annotation_added(annotation):
+    def _find_condition(resource):
+        node_annotations = resource.annotations.data_dict()
+
+        if annotation in node_annotations:
+            return True
+        else:
+            return False
+
+    return _find_condition
```

Comment:
Is this method check_annotation_added() used ? Didn't see a call

Reply:
Removed it